### PR TITLE
Remove unnecessary prints and String-Casts

### PR DESCRIPTION
**Pull request by MaZderMind**

### CODE REVIEW

File: python-api/gstswitch/controller.py [hunk 1]
```diff
@@ -295,8 +294,8 @@ def adjust_pip(self, xpos, ypos, width, height):
         try:
             conn = self.connection.adjust_pip(xpos, ypos, width, height)
             res = conn.unpack()[0]
-            print "adjust pip xpos:%s ypos:%s w:%s h:%s" % (
-                str(xpos), str(ypos), str(width), str(height))
+            print "adjust pip xpos:%u ypos:%u w:%u h:%u" % (
```
Comment (mithro) [inline python-api/gstswitch/controller.py]: I'm guessing these should actually be %i - but these prints should probably be removed anyway - see https://github.com/timvideos/gst-switch/issues/159

**Comment by MaZderMind** [comment]
I'm absolutely with you. The controller should not print to stdout anyway.

File: python-api/gstswitch/controller.py [hunk 2]
```diff
@@ -339,10 +329,8 @@ def click_video(self, xpos, ypos, width, height):
         try:
             conn = self.connection.click_video(xpos, ypos, width, height)
             res = conn.unpack()[0]
-            if res is True:
-                print "Click video: xpos:%s ypos:%s width:%s height:%s" % (
-                    str(xpos), str(ypos), str(width), str(height))
-            else:
+            if not res is True:
```
Comment (mithro) [inline python-api/gstswitch/controller.py]: Here you should use;
```python
if res is not True:
```

Coming from other languages you might expect the above to be equivalent to the following, but it instead does what the english says;
```python
if res is (not True):
```

**Comment by mithro** [comment]
Looks like landscape actually picked up on this "code smell" -> https://landscape.io/diff/81447

**Pull request merged**